### ISSUE TITLE
StreamingHttpResponse.streaming - missing attribute

### ISSUE THREAD

**Issue by jonfoster** [typeddjango/django-stubs#629]
# Bug report
## TLDR

The django.http.response.StreamingHttpResponse.streaming attribute is missing.  It's a bool.

## What's wrong

I get a false positive from mypy:

> foo.py:5: error: Item "StreamingHttpResponse" of "Union[HttpResponse, StreamingHttpResponse]" has no attribute "streaming"

Test case is:

```
from typing import Union
from django.http.response import HttpResponse, StreamingHttpResponse

def func(response: Union[HttpResponse, StreamingHttpResponse]) -> None:
    if response.streaming:
        pass
```

## How is that should be

Should not report an error!

## Obvious fix

This is a 1-line fix: In django-stubs/http/response.pyi, copy the definition of "streaming" from HttpResponse to StreamingHttpResponse.

## Better fix

However, a much better fix would be to move that definition to HttpResponseBase instead.  Part of the API of HttpResponseBase is that there is always a "streaming" attribute.  Django doesn't actually define it there, because without type annotations there's no point.

Moving the attribute definition to HttpResponseBase, would allow me to use "HttpResponseBase" as the type annotation, instead of the clunky "Union[HttpResponse, StreamingHttpResponse]".

**Comment by sobolevn** [comment]
@jonfoster please, go ahead! 👍
